File: 문제풀이/온라인 저지/백준/# 02. Bronze Ⅳ/# 10101. 삼각형 외우기/src/Main.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int a = Integer.parseInt(br.readLine()); // 각1
        int b = Integer.parseInt(br.readLine()); // 각2
        int c = Integer.parseInt(br.readLine()); // 각3

        br.close();
        String result;

        if (a+b+c == 180) { // 각의 합이 180인 경우

            if (a==b && b==c && c == a) { // 세 각의 크기가 같다
                result = "Equilateral";
            } else if (a==b || b==c || c==a) { // 그 외에, 적어도 두 각의 크기가 같다.
                result = "Isosceles";
            } else { // 그 외 : 서로 같은 각이 없는 경우
                result = "Scalene";
            }
        } else { // 각의 합이 180이 아닌 경우
            result = "Error";
        }

        System.out.println(result);
    }
}

File: 문제풀이/온라인 저지/백준/# 02. Bronze Ⅳ/# 10101. 삼각형 외우기/python/main.py
a, b, c = int(input()), int(input()), int(input())

if a + b + c != 180:
    answer = 'Error'
elif a == b == c:  # 정삼각형
    answer = 'Equilateral'
elif a == b or b == c or c == a:  # 이등변 삼각형
    answer = 'Isosceles'
else:
    answer = 'Scalene'

print(answer, end='')
